storefront/test/view_models/workarea/storefront/user_activity_view_model_test.rb

```diff
@@ -19,7 +18,10 @@ def reset_display_count
       def test_products
         product_one = create_product
         product_two = create_product
-        product_three = create_product
+        product_three = create_product(
```

Comment:
What do these have to do with the change? I'm not following...

Reply:
differentiates this product from the other two